mmv1/templates/terraform/examples/region_url_map_default_route_action.tf.erb

```diff
@@ -65,6 +65,46 @@ resource "google_compute_region_url_map" "<%= ctx[:primary_resource_id] %>" {
         response_headers_to_remove = ["buzz"]
       }
     }
+    url_rewrite {
+      host_rewrite = "A replacement header"
+      path_prefix_rewrite = "A replacement path"
```

Comment:
I worry about server side validation breaking this in the future since they aren't valid hosts/prefixes.

Reply:
Yeah that's a fair worry! I've replaced these values in 7613ca4